docs/release-notes/index.rst

```diff
@@ -0,0 +1,253 @@
+=============
+
+This page lists past release notes for Starcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Starcraft.
+
+
+Current releases
+----------------
+
+
+<latest release>
+~~~~~~~~~~~~~~~~
+
+- <link to latest release, update, such as 8.2.0>
+- <link to latest release, update, such as 8.1.0>
+- <link to latest release, initial, such as 8.0.0>
+
+
+<parallel release>
+~~~~~~~~~~~~~~~~~~
+
+<If necessary, add guidance and caveats about these older releases, such as
+"Snapcraft 7 is available for building core18 snaps. When building for a newer
+base, use Snapcraft 8.">
+
+- <link to parallel release, update, such as 7.2.0>
+- <link to parallel release, update, such as 7.1.0>
+- <link to parallel release, initial, such as 7.0.0>
+
+
+Past releases
+-------------
+
+
+<past release>
+~~~~~~~~~~~~~~
+
+- <link to past release, update, such as 6.2.0>
+- <link to past release, update, such as 6.1.0>
+- <link to past release, initial, such as 6.0.0>
+
+
+.. _release_policy_and_schedule:
+
+Release policy and schedule
+---------------------------
+
+Canonical is committed to supporting the <"latest major release" or "last two
+major releases"> of Starcraft. <Optional: "We forward-port changes in older
+releases to the latest release, if they're compatible.">
+
+Starcraft is released according to the Semantic Versioning 2.0.0 scheme with
+numbers for major, minor, and patch versions.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Example
+      - Significance
+    * - Major
+      - **3**.1.2
+      - A change that breaks compatibility with the previous version.
+    * - Minor
+      - 3.\ **1**\ .2
+      - A new feature within the major version.
+    * - Patch
+      - 3.1.\ **2**
+      - A bug fix within the major or minor version.
+
+Starcraft is released when it achieves development milestones in its product
+lifecycle. It doesn't follow a predefined release cadence.
+
+
+Long-term support
+-----------------
+
+Starcraft doesn't have long-term support (LTS) releases. However, we typically
+deliver a compatibility patch shortly after Ubuntu LTS releases to ensure
+continuity.
+
+.. toctree::
+   :maxdepth: 1
+
+
+.. release note template:
+
+  Starcraft 1.0 release notes
+  ===========================
+
+  15 October 2024
+
+  Learn about the new features, changes, and fixes introduced in Starcraft 1.0.
+  For information about the Starcraft release cycle, see the
+  :ref:`release_policy_and_schedule`.
+
+
+  Requirements and compatibility
+  ------------------------------
+
+  Starcraft 1.0 requires Python 3.11 or higher.
+
+  <If there are multiple requirements, remove "Python 3.11 or higher" in the
+  previous paragraph and add a separate list here, with the same format of
+  "<package> or higher".>
+
+  For development and testing, Starcraft requires at minimum a <architecture>
+  system or VM with <number>GB RAM.
```

Comment:
What do you envision for a "minimum" architecture?

Reply:
Good point. The minimum should be before the RAM. That sentence is generally open-ended. I imagine most of our hardware requirements are generally lax.

Reply:
[06d5514](https://github.com/canonical/starbase/pull/286/commits/06d5514f5aaebc8691059c6b563f6f8df7e0608d)

Reply:
Agreed.  That looks good to me